snap/snapcraft.yaml

```diff
@@ -200,6 +217,47 @@ parts:
       - libseccomp-dev
       - make
 
+  nvidia-container-toolkit:
+    plugin: go
+    source: https://github.com/NVIDIA/nvidia-container-toolkit.git
+    source-tag: v1.12.0
+    source-depth: 1
+    override-build: |
+      craftctl default
+
+      # From nvidia-container-toolkit package postinst
+      (cd "${CRAFT_PART_INSTALL}/bin" && ln -s nvidia-container-runtime-hook nvidia-container-toolkit)
```

Comment:
is this compat / legacy name? or does the binary behave different when called under different name?

Reply:
To be honest, I don't know the answer, as per my comment int eh part it's from the upstream package postinst.

I will try and find out.  If I find nothing useful about it, we can always just take it out and see what breaks.

Reply:
Turns out this was renamed at some point, and seems to be for compatibility.

Ref: https://github.com/NVIDIA/nvidia-container-toolkit/commit/8a56671d18a3f1953ea973b1b744bc9cb97b72b9

Therefor, as it;s internal to the snap at this point, it should be find to remove, which I've done.